diff -p -up ./CMakeLists.txt.orig ./CMakeLists.txt
--- ./CMakeLists.txt.orig	2012-07-23 03:33:19.000000000 +0800
+++ ./CMakeLists.txt	2012-07-26 11:45:22.557989729 +0800
@@ -52,7 +52,6 @@ IF(MANAGE_ENVIRONMENT_PATH STREQUAL "NOT
 ENDIF(MANAGE_ENVIRONMENT_PATH STREQUAL "NOTFOUND")
 
 INCLUDE(ManageVersion)
-RELEASE_NOTES_READ_FILE()
 INCLUDE(ManageUninstall)
 
 
@@ -114,12 +113,6 @@ ADD_SUBDIRECTORY(src bin)
 # Installing
 #
 
-SET(INSTALL_DOCS ${RELEASE_NOTES_FILE} AUTHORS README ChangeLog COPYING USER-GUIDE)
-STRING_JOIN(PRJ_DOC_LIST " " ${INSTALL_DOCS})
-
-INSTALL(FILES ${INSTALL_DOCS}
-    DESTINATION "${PRJ_DOC_DIR}")
-
 INSTALL(DIRECTORY icons
     DESTINATION  ${PRJ_DATA_DIR})
 
@@ -135,69 +128,3 @@ INCLUDE(ManageGConf)
 #
 INCLUDE(ManageSourceVersionControl)
 MANAGE_SOURCE_VERSION_CONTROL_GIT()
-
-####################################################################
-# Packing
-#
-INCLUDE(ManageArchive)
-
-IF(CMAKE_FEDORA_ENABLE_FEDORA_BUILD)
-    INCLUDE(ManageRPM)
-    INCLUDE(ManageReleaseFedora)
-ENDIF(CMAKE_FEDORA_ENABLE_FEDORA_BUILD)
-INCLUDE(ManageRelease)
-
-SET(PRJ_IGNORE_FILES_COMMON
-    "/doc/"  "\\\\.spec$" "messages.po$" "\\\\.orig$" "/cmake-fedora/"
-    "/\\\\.cache/"
-    )
-
-# GOB_GENERATED source codes are now excluded,
-# As Bug 519108 (https://bugzilla.redhat.com/show_bug.cgi?id=519108)
-# Is fixed from Fedora 11
-
-LIST(APPEND PRJ_IGNORE_FILES_COMMON
-    "src/maker-dialog.*\\\\.[ch]"
-    "src/ibus-chewing-engine.*\\\\.[ch]"
-    "/SPECS/" "/BUILD/" "/BUILDROOT/" "/SOURCES/" "/SRPMS/" "/RPMS/"
-    "\\\\.swo$" "\\\\.rej$"
-    "/FedPkg/" "/\\\\."
-    )
-
-LIST(APPEND SOURCE_ARCHIVE_IGNORE_FILES
-    ${PRJ_IGNORE_FILES_COMMON} "/bin/" "\\\\.xml$" "\\\\.schemas")
-
-IF(RPM_BUILD_SOURCES)
-    SET(SOURCE_ARCHIVE_DIR ${RPM_BUILD_SOURCES})
-ELSE(RPM_BUILD_SOURCES)
-    SET(SOURCE_ARCHIVE_DIR ${CMAKE_BINARY_DIR}/SOURCES)
-ENDIF(RPM_BUILD_SOURCES)
-
-PACK_SOURCE_ARCHIVE("${SOURCE_ARCHIVE_DIR}")
-IF(CMAKE_FEDORA_ENABLE_FEDORA_BUILD)
-    PACK_RPM()
-    RELEASE_FEDORA(fedora)
-    # "Off" Warning is displayed if mock is not installed.
-    RPM_MOCK_BUILD()
-    MANAGE_RELEASE(release_fedora)
-ENDIF(CMAKE_FEDORA_ENABLE_FEDORA_BUILD)
-
-#####################################################################
-# Hosting and Release
-#
-
-#INCLUDE(ManageUpload)
-#ADD_CUSTOM_TARGET(upload
-#    COMMENT "Uploading all files"
-#    )
-
-#MANAGE_UPLOAD_CMD(scp src UPLOAD_FILES ${SOURCE_ARCHIVE_FILE})
-#ADD_DEPENDENCIES(upload upload_src)
-
-
-####################################################################
-# Test Suites.
-#
-ENABLE_TESTING()
-
-
